main.go

```diff
@@ -43,35 +55,57 @@ func main() {
 			cmd.MarkFlagRequired("name")
 			cmd.ParseFlags(args)
 
-			argConf := config.Config{App: *app, Stack: *stack, Stage: *stage}
-			conf, err := config.Read(argConf, config.DefaultFiles()...)
-			check(logger, err, "Unable to read config", InvalidArgs)
-
-			ssm := store.NewSSM(logger, ssmClient(context.TODO(), logger, *profile))
+			check(logger, configErr, "Unable to read config", InvalidArgs)
 
-			service := store.Service{App: conf.App, Stack: conf.Stack, Stage: conf.Stage}
-			item, err := ssm.Get(service, *name)
+			item, err := ssmStore.Get(service, *name)
 			check(logger, err, fmt.Sprintf("unable to get %s for service '%s'", *name, service.Prefix()), 1)
 
 			logger.Infof(item.String())
 		},
 	}
 
+	getSecretCmd := &cobra.Command{
```

Comment:
I wonder whether we should use a flag to denote retrieval from Secrets Manager rather than having a new set of commands?